### PR TITLE
adding travis for pep8

### PR DESCRIPTION
**Pull request by eolivi**

### CODE REVIEW

**Comment by agramfort** [comment]
why is travis not running?

let me try to close and open

**Comment by agramfort** [comment]
ok it's starting good !

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.7"
+
+install:
+# - python setup.py install
+  - pip install .
```
Comment (agramfort) [inline .travis.yml]: remove this line

**Comment by agramfort** [comment]
then travis should be happy

**Comment by eolivi** [comment]
travis is not happy, but thats normla, because the code in this PR is not pep8 compliant

**Comment by agramfort** [comment]
how about making it pep8 then :)

**Comment by eolivi** [comment]
that's the other PR https://github.com/openmeeg/openmeeg_sample_data/pull/5 ...

**Comment by agramfort** [comment]
good point :)

**Pull request merged**